superset-frontend/webpack.config.js

```diff
@@ -283,7 +283,9 @@ const config = {
   },
   resolve: {
     modules: [APP_DIR, 'node_modules', ROOT_DIR],
-    alias: {},
+    alias: {
```

Comment:
leave a todo comment, notice we will remove `alias` after upgrade react 17

Reply:
Good idea 👍